support/reftest/reftest.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env python3
```

Comment:
This file will need to be deleted now that we have /reftest, yeah?

Reply:
@nathanegillett  deleted.